tools/bash.py

```
@@ -1,257 +1,230 @@
-        
+
         try:
             exec(script_code)
             output_out = redirected_output.getvalue()
             error_out = redirected_error.getvalue()
-            
-            # Save successful code
+            display.add_message("user", f"Output:\n{output_out}")
+            display.add_message("user", f"Error Message:\n{error_out}")
             if not error_out:
                 saved_path = save_successful_code(script_code)
```

Ruff (0.8.2)

Local variable `saved_path` is assigned to but never used

Remove assignment to unused variable `saved_path`

(F841)